systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java

```diff
@@ -28,6 +29,7 @@ public static long getTimeoutForResourceReadiness(String kind) {
 
         switch (kind) {
             case Kafka.RESOURCE_KIND:
+            case KafkaNodePool.RESOURCE_KIND:
```

Comment:
We have 14 minutes for Kafka resource because of the ZK, Kafka, and EO pods -> maybe the KafkaNodePool resource can depend on number of replicas that are passed to the resource? Or something like 5 minutes should be sufficient?